### ISSUE TITLE
[FEATURE] remove the unnecessary clutter in home screen 

### ISSUE THREAD

**Issue by rajarshikhatua** [anonfaded/FadCam#81]
for inspiration the home screen should look like any of these two app https://play.google.com/store/apps/details?id=com.blackmagicdesign.android.blackmagiccam&hl=en_IN

or 

https://play.google.com/store/apps/details?id=com.filmic.filmicpro&hl=en_IN

with the current layout you can't adjust exposer or focus it's purely aimed for hidden and emergency use but i want to be it a feature packed camera app not just for one purpose but suitable for everything

**Comment by anonfaded** [comment]
Alright this is a good feature request, but this will need alot of refactoring as it will change many ui elements and will need more code to have all the control features. This can take alot of time but adding it to the list of upcoming features.

**Comment by psyburr** [comment]
Let's stay focused on the "mission," such as this text from the README,
> _Seamless_ **background** video recorder for Android

 I do not disagree with the idea that having a manual adjustment for a few settings if the user chooses to activate them, however; making that the default may very well negate the _seamless_ part. Including every camera function the device is capable of certainly strays far away from the purpose of being a background camera. A user ought be able to open the app and have it be recording in as few steps as possible, _after a brief initial configuration_, and continue recording when switching to another app or locking/turning off the screen.

**Comment by anonfaded** [comment]
> making that the default may very well negate the seamless part.

Yeah i understand, the camera adjustment settings like zooming etc won't be a default thing for sure. This all will be additional just for those who actually want it else it will remain disabled. The main purpose of the app won't be compromised.

**Comment by rajarshikhatua** [comment]
This is completely unrelated, but since Open Camera is [open source](https://sourceforge.net/p/opencamera/code) under the GPL license, is it possible to patch it to support background video recording?

**Comment by anonfaded** [comment]
> This is completely unrelated, but since Open Camera is [open source](https://sourceforge.net/p/opencamera/code) under the GPL license, is it possible to patch it to support background video recording?

yes it is possible, but needs a lot of refactoring. I forked that, will try someday when i have time

**Comment by rajarshikhatua** [comment]
> yes it is possible, but needs a lot of refactoring. I forked that, will try someday when i have time

if you have to compare between -
1. making fadcam a full fledged camera app
2. making an already great camera app (might be other than open camera) work in background

which is more harder!

**Comment by anonfaded** [comment]
> which is more harder!

refactoring fadcam at the moment is harder bcz of the design of the app.

There is difference in the purpose of both apps,  other camera apps are designed for photography while fadcam is more focused on off-screen recording feature to capture important moments. And to add normal camera features to fadcam, it needs more refactoring, mainly with the preview feature where i need to show preview but also make it responsive with features like zoom contorl or exposure etc. It is hard but not impossible, currently i'm stuck in implementing the openGL feature where i am facing issues with showing the preview so i need to fix this, and this opengl setup will make it easier to add other camera features like exposure control etc.

**Comment by rajarshikhatua** [comment]
sounds promising
